etc/vspc.conf.sample

```diff
@@ -6,4 +6,4 @@ cert = cert.pem
 key = key.pem
 uri = vmware-vspc
 serial_log_dir = /opt/vmware/vspc
-
+log_file_size = 1073741824
```

Comment:
I'm just wondering, are you agree to use 1GB for the threshold to start to resize the log file?

Reply:
This is just an example, right? I wouldn't same more than 50MB per VM in production. Even that seems quite big to me.

Reply:
yes